### DISCUSSION TITLE
Dropdown for TOC sections?

### DISCUSSION THREAD

**Discussion by richardliaw** [executablebooks/meta#597] [Q&A]
Hi all, 

Thanks for all the hard work you've put into this great project! 

I'm part of the ray project(https://docs.ray.io/en/latest/) , where we are users of the sphinx-book-theme. We are currently on 0.0.42, and are looking to upgrade both Sphinx and the Book theme. 

In the 0.0.42 version of the project, we are able to configure dropdowns for the sidebar. 

<img width="345" alt="Screen Shot 2021-12-27 at 1 54 21 PM" src="https://user-images.githubusercontent.com/4529381/147509584-3affa156-1816-439e-b47f-ed7d3bf241f5.png">

However, after upgrading to latest Sphinx and the Book theme, it seems like this is no longer supported: https://ray--21266.org.readthedocs.build/en/21266/

Is there a way to get this to work again?

**Comment by AakashGfude** [comment]
Hi @richardliaw, yes this is an old feature. @choldgraf this top-level collapsible feature is not available at the moment, right? Maybe we can consider adding it using a config variable?

**Reply by choldgraf** [reply]
Maybe we could support the value `0` here: https://sphinx-book-theme.readthedocs.io/en/latest/customize/sidebar-primary.html#control-the-depth-of-the-left-sidebar-lists-to-expand and this would then make the captions have a dropdown arrow?

**Comment by choldgraf** [comment]
Can you go into more detail on what you mean by "configure dropdowns"? Are you trying to make the dropdowns display the sub-pages by default? Or are you talking about making documentation **sections** be configurable with dropdowns?

**Reply by richardliaw** [reply]
Yeah, I'm talking about making the documentation *sections* configurable with dropdowns. I'd like it to default to this:

<img width="317" alt="Screen Shot 2021-12-29 at 1 37 18 PM" src="https://user-images.githubusercontent.com/4529381/147705258-2ca6532a-12e0-4ab4-98b2-fb5708940960.png">

**Reply by AakashGfude** [reply]
I think we should have the default as what we have at the moment which would be `navbar_collapsible_level: 1`, and have the possibility to configure it to the above image. setting something like `navbar_collapsible_level: 0`.

**Reply by richardliaw** [reply]
yes, that would be awesome!

**Comment by richardliaw** [comment]
Hi @choldgraf @AakashGfude any update on this? A `navbar_collapsible_level` feature would be awesome.

**Reply by choldgraf** [reply]
Nope - people are working on a lot of other stuff right now. However I've created an issue to track this feature request with some rough guidelines to implement here: https://github.com/executablebooks/sphinx-book-theme/issues/460

if you're interested in giving a shot at implementation, I am happy to review a PR!
